README.md

```diff
@@ -71,7 +71,7 @@ More options for `runner.py`
                         (default: dummy_samples.json)
   --year YEAR           Year
   --campaign CAMPAIGN   Dataset campaign, change the corresponding correction
-                        files{ "Rereco17_94X","Winter22Run3","2018_UL","2017_UL","2016preVFP_UL","2016postVFP_UL"}
+                        files{ "Rereco17_94X","Winter22Run3","Summer22Run3","Summer22Run3EE","2018_UL","2017_UL","2016preVFP_UL","2016postVFP_UL"}
```

Comment:
Hi @Ming-Yan just a minor comment: could you change the name of the `Summer22Run3EE` to `Summer22EERun3` to be consistent with what is used as names in the MC campaigns?

Reply:
Done! and also add new JEC files

Reply:
These changes look good :) do we have btw the jet veto maps included somewhere? From their recommendations, it seems like they are recommended for the Summer22 Riun3 campaigns

Reply:
Oops, I check it at some point and not integrated to the framework. I need to add this back